### PR TITLE
Issue 3 user can edit an entry

### PR DESCRIPTION
**Pull request by David-Gonzalez11**
https://user-images.githubusercontent.com/24907191/172067604-2a90ace9-e196-49a2-b530-cb66a6a8ad28.mov

### CODE REVIEW

**Comment by David-Gonzalez11** [comment]
https://user-images.githubusercontent.com/24907191/172269729-1319c4d4-0669-4df9-a88a-52ff27dada9d.mov

File: js/main.js [hunk 1]
```diff
@@ -47,41 +80,63 @@ function renderEntry(entry) {
...
+    var existingEntryId = document.querySelector('#existingEntryId');
+    $title.value = (data.editing.title);
+    $notes.value = (data.editing.notes);
+    $photoUrl.value = (data.editing.photo);
+    img.setAttribute('src', $photoUrl.value);
+    existingEntryId.value = (data.editing.id);
+  }
+}
+window.addEventListener('DOMContentLoaded', DOMContentLoaded);
 function DOMContentLoaded(event) {
```
Comment (BrettAlbright) [inline js/main.js]: This function should be lowercase, only Classes in JS are typically capitalized.

File: js/main.js [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
-$newBtn.addEventListener('click', viewEntryForm);
+$newBtn.addEventListener('click', newButton);
 var noEntries = document.querySelector('h5');
-// var EntriesText = document.querySelector('.entries-text');
+var img = document.querySelector('img');
+var input = document.querySelector('#photoUrl');
+input.addEventListener('input', handleInput);
+var $photoUrl = document.querySelector('.image');
+$photoUrl.addEventListener('input', handleInput);
+var EntriesText = document.querySelector('.entry-text');
```
Comment (BrettAlbright) [inline js/main.js]: This variable should start with a lowercase letter.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
2 tiny casing updates!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**